### PR TITLE
Chapter 4

### PR DESCRIPTION
**Pull request by leducson**
selection_005

### CODE REVIEW

**Comment by leducson** [comment]
ready

**Comment by hhhhuy112** [comment]
conflicted
Fix conflict

**Comment by leducson** [comment]
ready

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<% provide :title, t(".title") %>
-<h1><%= t(".title") %></h1>
+<h1><%= t(".title")%></h1>
```
Comment (hhhhuy112) [inline app/views/static_pages/home.html.erb]: <%= t ".title" %>

**Comment by hhhhuy112** [comment]
commented

**Comment by leducson** [comment]
ready

**Comment by hhhhuy112** [comment]
merged
You can merge it

**Pull request merged**